### DISCUSSION TITLE
Psd background from Transparent to White

### DISCUSSION THREAD

**Discussion by Alex2981** [dlemstra/Magick.NET#1516] [Help]
Hello!
when I open a psd with transparent background and save it to another file the background change to white:

using (var image = new MagickImage("myfile.psd", MagickFormat.Psd))
{
    image.Write("out.psd", MagickFormat.Psd);
}

a sample file is attached [psd.zip](https://github.com/dlemstra/Magick.NET/files/13885840/psd.zip)

thanks!

**Comment by dlemstra** [comment]
~~It looks like this is happening because your layer has a white background?~~

I did not notice you used `MagickImage` instead of `MagickImageCollection`. Switching to that should get you the output image that you want.

**Reply by Alex2981** [reply]
thanks, it works!
the output has a single layer image, is there a way to preserve the mask?
